firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -938,9 +970,11 @@ def pbs_for_update(client, document_path, field_updates, option):
     """
     if option is None:
         # Default uses ``exists=True``.
-        option = client.write_option(create_if_missing=False)
+        option = client.write_option(exists=True)
 
     transform_paths, actual_updates = remove_server_timestamp(field_updates)
+    if not actual_updates:
```

Comment:
It's fine to only specify server timestamps. There should be either `transform_paths` or `actual_updates`.

Reply:
Done.  Is this ValueError check needed for pbs_for_set also?

Reply:
If you update to the latest version of the conformance tests, you will no longer need it for set. Both set({}) and set({}, merge:true) should be allowed.